### ISSUE TITLE
Trained disparity maps have no negative values?

### ISSUE THREAD

**Issue by tianshenghui** [gengshan-y/high-res-stereo#12]
I trained this model on remote sensing images. The basic contour of the disparity map is correct, but the positions that should be negative on the disparity map are all extremely small numbers greater than 0.

**Comment by gengshan-y** [comment]
Are you talking about negative disparities? This method only searches for correspondences in one direction. If so, you could consider using optical flow or other forms of correspondence models.

**Comment by tianshenghui** [comment]
@gengshan-y I have solved this problem. In the original code, the features were operated by F.softmax before which were fed into diaparity regression module , I substitute it with F.normalize
